hydrocron/db/io/swot_shp.py

```diff
@@ -66,6 +66,7 @@ def read_shapefile(filepath, obscure_data, columns, s3_resource=None):
         shp_file = gpd.read_file('zip://' + filepath)
         with zipfile.ZipFile(filepath) as archive:
             shp_xml_tree = ET.fromstring(archive.read(filename[:-4] + ".shp.xml"))
+            logging.info(shp_xml_tree)
 
     numeric_columns = shp_file[columns].select_dtypes(include=[np.number]).columns
```

Comment:
Can this:

numeric_columns = shp_file[columns].select_dtypes(include=[np.number]).columns

be moved under the if obscure_date statement? Although I think this is trivial, if it is only needed for obscuring data then the column data type look up is only run when obscuring data.

Reply:
done